### PR TITLE
new car added

### PR DESCRIPTION
**Pull request by pete85**
## changes

- new car added
- car called 567

### CODE REVIEW

**Review by mugshepherd — CHANGES_REQUESTED** [review]
Please add additional information @pete85

**Review by bayramcura — COMMENTED** [review]
This test did not fully do ...., could you please review the code in car567.md file.  You may want to....

**Review by bayramcura — CHANGES_REQUESTED** [review]
New request change for file.....

File: cars/car567.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is now car567.md
```
Comment (mugshepherd) [inline cars/car567.md]: Piotr - would you please add additional information here?

Comment (mugshepherd) [reply cars/car567.md]: VIN number, make, model, etc.

Comment (mugshepherd) [inline cars/car567.md]: additional comment...

**Review by mugshepherd — CHANGES_REQUESTED** [review]
this looks relatively good but would you please make the requested changes.

File: cars/car567.md (hunk 1 above)
Comment (pete85) [reply cars/car567.md]: More details added

Comment (pete85) [reply cars/car567.md]: done

**Pull request closed without merging**